web/src/features/Dashboard/components/DashboardVersionCard.tsx

```diff
@@ -258,12 +265,7 @@ const DashboardVersionCard = (props: Props) => {
       <div className="flex action-tab-bar u-marginTop--10">
         {tabs
           .filter((tab) => tab !== "renderError")
-          .filter((tab) => {
-            if (isHelmManaged) {
-              return tab.startsWith("helm");
-            }
-            return true;
-          })
+          .filter((tab) => filterNonHelmTabs(tab, isHelmManaged))
```

Comment:
I think this should be filtered in the API but we'll do a FE fix for this for now. basically we take this logic and apply it to the initial tab selection.